script/openqa-bootstrap

```diff
@@ -6,6 +6,14 @@ set -x
 dbname="${dbname:="openqa"}"
 dbuser="${dbuser:="geekotest"}"
 
+
+# ensure that systemd is available (which is not the case in some containers)
+if ! systemctl > /dev/null ; then
```

Comment:
It can still be useful to run the script within a container. Even if the systemd-specific parts in the end don't work it would make sense to be able to run the first parts. What about:

```suggestion
if [[ -n $BYPASS_SYSTEMD_CHECK ]] && ! systemctl > /dev/null ; then
```